sdk/python/sawtooth_sdk/client/config.py

```diff
@@ -82,12 +83,21 @@ def _get_log_config(filename=None):
         log_config (dict): The dictionary to pass to logging.config.dictConfig
     """
     if filename is not None:
+        try:
```

Comment:
This would split a filename like "validator.logging.file.yaml" into [validator, logging, file, yaml] and use 'logging' string as the ext.  Better to just do filename.endswith(".yaml") check.